api/Element.json

```
@@ -10288,6 +10288,48 @@
           }
         }
       },
+      "setHTML": {
+        "__compat": {
+          "spec_url": "https://wicg.github.io/sanitizer-api/#dom-element-sethtml",
+          "support": {
+            "chrome": {
+              "version_added": false,
+              "notes": "Version 105 to 118 (inclusive) supported this method with a significantly different specification."
```

For mirroring, it's actually safer to prefix every version by the browser name:

```suggestion
              "notes": "Chrome 105 to Chrome 118 (inclusive) supported this method with a significantly different specification."
```